src/documents/README.md

```diff
@@ -5,66 +5,82 @@ layout: default
 
 ## Infusion
 
+### Writing Configuration
+* [Understanding Infusion Components](UnderstandingInfusionComponents.md)
+* [Understanding Component Options And Their Defaults](ComponentOptionsAndDefaults.md)
+* [Component Configuration Options](ComponentConfigurationOptions.md)
+* [Component Grades](ComponentGrades.md)
+* [Function Grades](FunctionGrades.md)
+* [Options Merging](OptionsMerging.md)
+* [Component Lifecycle](ComponentLifecycle.md)
+* [Priorities](Priorities.md)
 * [Framework Concepts](FrameworkConcepts.md)
-* Components
-  * [Understanding Infusion Components](UnderstandingInfusionComponents.md)
-  * [Understanding Component Options And Their Defaults](ComponentOptionsAndDefaults.md)
-  * [Component Configuration Options](ComponentConfigurationOptions.md)
-  * [Component Grades](ComponentGrades.md)
-  * [Options Merging](OptionsMerging.md)
-  * [Component Lifecycle](ComponentLifecycle.md)
-* [DOM Binder](DOMBinder.md)
-* Inversion of Control
-  * [How To Use Infusion IoC](HowToUseInfusionIoC.md)
-  * [Subcomponent Declaration](SubcomponentDeclaration.md)
-  * [Contexts](Contexts.md)
-  * [Invokers](Invokers.md)
-  * [Expansion Of Component Options](ExpansionOfComponentOptions.md)
-  * [IoCSS](IoCSS.md)
-  * [Declarative this-ism In IoC](DeclarativeThisismInIoC.md)
-  * [IoC References](IoCReferences.md)
-* Events
-  * [Infusion Event System](InfusionEventSystem.md)
-  * [Event injection and boiling](EventInjectionAndBoiling.md)
-* ChangeApplier
-  * [ChangeApplier](ChangeApplier.md)
-  * [ChangeApplier API](ChangeApplierAPI.md)
-* [Model Relay](ModelRelay.md)
 * [Progressive Enhancement](ProgressiveEnhancement.md)
```

Comment:
Ah, yes, actually... there is no Progressive Enhancement anymore; replaced by Context Awareness. Removing link.